resources/benchmark-runner.mjs

```diff
@@ -528,6 +540,8 @@ export class BenchmarkRunner {
             const values = [];
             for (const suiteName in this._measuredValues.tests) {
                 const suiteTotal = this._measuredValues.tests[suiteName].total;
+                if (suiteTotal <= 0)
```

Comment:
Instead of waiting until the very end, we should check `this._measuredValues.tests[suite.name].total` is non-zero at the end of `_runSuite`.

Reply:
Added `_validateSuiteTotal` which is called from `_runSuite`